December news (#695)
ropensci/roweb3: content/blog/2023-12-22-news-december-2023/index.Rmd

---
title: 'rOpenSci News Digest, December 2023'
author: 'The rOpenSci Team'
date: '2023-12-22'
tags:
  - newsletter
slug: news-december-2023

> description: donate to rOpenSci; deposits package for data publishing; a package a day; new packags and package news

Reviewer: ```suggestion
description: donate to rOpenSci; deposits package for data publishing; a package a day; new packages and package news
```

    return(paste(c(...), collapse = " and "))
  }
  glue::glue_collapse(..., sep = ", ", last = ", and ")
}
```
<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->

Dear rOpenSci friends, it's time for our monthly news roundup!
<!-- blabla -->
You can read this post [on our blog](`r url`).
Now let's dive into the activity at and around rOpenSci!

## rOpenSci HQ

### Reminder about giving season: consider donating to rOpenSci

This Giving Season consider donating to rOpenSci to support our mission of empowering Open Science.

Sustaining an open project with quality infrastructure freely accessible to the global and diverse community of R software users, research software developers, and engineers requires many different resources. Our organization’s ongoing costs are supported by grants and donations from individuals and organizations which share our vision and mission.

By supporting us, you’re not just donating; you’re contributing to a community that’s breaking down barriers, creating opportunities, and shaping the future of open and reproducible science for everyone. 

Join us in this vital mission today! You can donate here: https://ropensci.org/donate/

> Thanks to those who already donated!

Reviewer 2: ```suggestion
We are gratful for the donation we already receive. 
```

### Easier data publishing: An interview of Mark Padgham about the deposits package!

rOpenSci Mark Padgham was interviewed on the R Consortium blog about his [deposits package](https://www.r-consortium.org/blog/2023/11/30/deposits-r-package-delivers-a-common-workflow-for-r-users) that simplifies data sharing.

> "Publicly depositing datasets associated with published research is becoming more common, partly due to journals increasingly requiring data sharing and partly through more general and ongoing cultural changes with data sharing. Yet data sharing is often seen as time-consuming, particularly to meet the expectations of individual data repositories. While documentation and training can help familiarize users with data-sharing processes, browser-based data, and metadata submission workflows can only be so fast, are not easily reproduced, and do not facilitate regular or automated data and metadata updates. Better programmatic tools can transform data sharing from a mountainous climb into a pit of success."

This interview was furthermore featured on the [R Weekly highlights podcast](https://rweekly.fireside.fm/145?t=848) hosted by Eric Nantz and Mike Thomas.

> ### "A package a day" series on Mastodon!

Reviewer: And weekly package digests on LinkedIn?

Reviewer 2: Link to one of the post on LinkedIn: https://www.linkedin.com/feed/update/urn:li:activity:7140023562223943680

We've started sharing about one package a day from our federated community of open source software developers, on [Mastodon](https://mastodon.social/@rOpenSci@hachyderm.io/111522245481433695)!
Follow us on Mastodon, or search for `from:@rOpenSci@hachyderm.io #apackageaday` to find all posts until now.

### Resources for [multilingual publishing](/commcalls/nov2023-multilingual/) community call

> On this [community call](/commcalls/nov2023-multilingual/), [Maëlle Salmon](/author/ma%C3%ABlle-salmon/), [Paola Corrales](/author/paola-corrales/), and [Elio Campitelli](/author/elio-campitelli/), shared the rOpenSci Multilingual project details. Maëlle presented the R packages that allow us to have our content in several languages. Elio and Paola shared about the translation workflow and showed the Translation Guide written to document the process.

Reviewer: ```suggestion
On this [community call](/commcalls/nov2023-multilingual/), [Maëlle Salmon](/author/ma%C3%ABlle-salmon/), [Paola Corrales](/author/pao-corrales/), and [Elio Campitelli](/author/elio-campitelli/), shared the rOpenSci Multilingual project details. Maëlle presented the R packages that allow us to have our content in several languages. Elio and Paola shared about the translation workflow and showed the Translation Guide written to document the process.
```

- **[rvertnet](https://cran.r-project.org/web/packages/rvertnet/index.html)**, Retrieve, map and summarize data from the VertNet.org archives (<https://vertnet.org/>). Functions allow searching by many parameters, including taxonomic names, places, and dates. In addition, there is an interface for conducting spatially delimited searches, and another for requesting large datasets via email. [Issue for volunteering](https://github.com/ropensci-archive/rvertnet/issues/71).

### Call for co-maintainers

Refer to our somewhat [recent blog post](/blog/2022/10/17/maintain-or-co-maintain-an-ropensci-package/#packages-looking-for-co-maintainers) to identify other packages where help is especially wished for!
See also our [help wanted page](/help-wanted/) -- before opening a PR, we recommend asking in the issue whether help is still needed.

## Package development corner

Some useful tips for R package developers. :eyes:

### An older post about dependencies

Do you know of the post ["Playing on the same team as your dependency"](https://www.tidyverse.org/blog/2022/09/playing-on-the-same-team-as-your-dependecy/) by Thomas Lin Pedersen?

> It's a take from the maintainer perspective, on how maintainers of reverse dependencies can help make the relationship two-ways.

Reviewer: ```suggestion
It's a take from the maintainer perspective, on how maintainers of reverse dependencies can help make the relationship a two-way one.
```

  glue::glue_collapse(..., sep = ", ", last = ", and ")
}
```
<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->

Dear rOpenSci friends, it's time for our monthly news roundup!
<!-- blabla -->
You can read this post [on our blog](`r url`).
Now let's dive into the activity at and around rOpenSci!

## rOpenSci HQ

### Reminder about giving season: consider donating to rOpenSci

This Giving Season consider donating to rOpenSci to support our mission of empowering Open Science.

Sustaining an open project with quality infrastructure freely accessible to the global and diverse community of R software users, research software developers, and engineers requires many different resources. Our organization’s ongoing costs are supported by grants and donations from individuals and organizations which share our vision and mission.

By supporting us, you’re not just donating; you’re contributing to a community that’s breaking down barriers, creating opportunities, and shaping the future of open and reproducible science for everyone. 

Join us in this vital mission today! You can donate here: https://ropensci.org/donate/

We are grateful for the donations we have already received. 

> ### Easier data publishing: An interview of Mark Padgham about the deposits package!

Author: @mpadge ok with this summary?

rOpenSci Mark Padgham was interviewed on the R Consortium blog about his [deposits package](https://www.r-consortium.org/blog/2023/11/30/deposits-r-package-delivers-a-common-workflow-for-r-users) that simplifies data sharing.

> "Publicly depositing datasets associated with published research is becoming more common, partly due to journals increasingly requiring data sharing and partly through more general and ongoing cultural changes with data sharing. Yet data sharing is often seen as time-consuming, particularly to meet the expectations of individual data repositories. While documentation and training can help familiarize users with data-sharing processes, browser-based data, and metadata submission workflows can only be so fast, are not easily reproduced, and do not facilitate regular or automated data and metadata updates. Better programmatic tools can transform data sharing from a mountainous climb into a pit of success."

This interview was furthermore featured on the [R Weekly highlights podcast](https://rweekly.fireside.fm/145?t=848) hosted by Eric Nantz and Mike Thomas.

> 

Reviewer 2: ```suggestion
## rOpenSci Developer Guide Translation to Portuguese 

We want to express our gratitude to the community members who have been actively contributing to the localization and translation of the rOpenSci Developer Guide into Portuguese. Your dedication and efforts are making this valuable resource accessible to an even wider audience.  Here's a quick update on the progress:

*    18% of the book is already translated! 📚
*    2 chapters are awaiting a second review. 👀
*    24 chapters are still awaiting translation. 🌐
*    7 amazing contributors are driving this initiative: _Pedro Duarte Faria, Daniel Vartanian, Marcelo Perlin, Beatriz Milz, João Granja-Correia, António Pedro and Mariana._ 👏  Thank you for the incredible work you've done so far! 🙌

We're not done yet, and we invite more members of our community to join this collaborative effort. If you want to contribute to the translation project, join us on GitHub: [rOpenSci Developer Guide Translation Project](https://github.com/orgs/ropensci/projects/7). 🚀

```

Reviewer 2: My intention is to have some chapter for the first review ready before we sent the newsletter. If that is the case, then some of the number can change.
